src/main/java/Duke.java

```diff
@@ -1,10 +1,69 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        String input;
+        ArrayList<Task> tasks = new ArrayList<>();
+        while (true) {
+            input = scanner.nextLine();
+            switch (input) {
+                case "bye":
+                    System.out.println("Bye. Hope to see you again soon!");
+                    return;
+                case "list":
+                    for (int i = 0; i < tasks.size(); i++) {
+                        System.out.printf("%d.%s\n", i + 1, tasks.get(i));
+                    }
+                    break;
+                default: // changes made
```

Comment:
The default block in this switch expression is a bit long and confusing. Would it be more readable if you used one case for each command?